app/lib/frontend/templates.dart

```diff
@@ -835,10 +861,12 @@ String _getAuthorsHtml(List<String> authors) {
   }).join('<br/>');
 }
 
+bool _isAnalysisSkipped(AnalysisStatus status) =>
```

Comment:
Maybe skipped is the wrong word here? I was thinking of packages that are brand new (no data yet), or there's a minor problem why the data isn't there right now; not so much that it's packages that are obsolete.

Reply:
We don't have an `AnalysisStatus` value for the pending ones (e.g. the one that is scheduled but not done yet), only for the ones that have been done. This way if a status is missing (==null), then it is by default pending (and should be scheduled), otherwise it is done.

We 'skip' two kinds of packages in the analysis: one that is older than two years, and one that is flagged as discontinued. We shall analyse everything else.

Ideally https://github.com/dart-lang/pub-dartlang-dart/issues/1352 could also track the scheduled but not done yet status too.